src/newSeed/stage4.ts

```diff
@@ -90,17 +90,17 @@ export class Stage4 extends BaseStage {
     if (!this.seedConfig.seedDetails.pricePerToken || this.seedConfig.seedDetails.pricePerToken === "0") {
       message = "Please enter a value for Funding Tokens per Seed Token";
     } else if (!this.seedConfig.seedDetails.fundingTarget || this.seedConfig.seedDetails.fundingTarget === "0") {
-      message = "Please enter a non-zero value for the Funding Target";
+      message = "Please enter a number greater than zero for the Funding Target";
     } else if (!this.seedConfig.seedDetails.fundingMax || this.seedConfig.seedDetails.fundingMax === "0") {
-      message = "Please enter a non-zero number for the Funding Max";
-    } else if (!(Number(this.seedConfig.seedDetails.vestingDays) > 0)) {
+      message = "Please enter a number greater than zero for the Funding Max";
+    } else if (!(this.seedConfig.seedDetails.vestingDays > 0)) {
       message = "Please enter a number greater than zero for  \"Seed tokens vested for\" ";
-    } else if (!(Number(this.seedConfig.seedDetails.vestingCliff) > 0)) {
-      message = "Please enter a number greater than zero for \"with a cliff of\" ";
-    } else if (BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)) {
-      message = "Please enter a value for Funding Target smaller than Funding Max";
+    } else if (!(this.seedConfig.seedDetails.vestingCliff >= 0)) {
+      message = "Please enter a value for \"with a cliff of\" ";
     } else if (this.seedConfig.seedDetails.vestingCliff > this.seedConfig.seedDetails.vestingDays) {
       message = "Please enter a value of \"with a cliff of\" less than \"Seed tokens vested for \"";
+    } else if (BigNumber.from(this.seedConfig.seedDetails.fundingTarget).gt(this.seedConfig.seedDetails.fundingMax)) {
+      message = "Please enter a value for Funding Target smaller than Funding Max";
```

Comment:
can you please confirm whether target and max can be equal?  I don't see why not.

Reply:
it can

Reply:
ok, can you please correct this validation accordingly?

Reply:
This validation would allow for the 2 values to be equal

Reply:
This:

> "Please enter a value of \"with a cliff of\" less than \"Seed tokens vested for \""

Should read:

> "Please enter a value of \"with a cliff of\" less than or equal to \"Seed tokens vested for \""

Reply:
This:

> "Please enter a value for Funding Target smaller than Funding Max"

Should read:

> "Please enter a value for Funding Target less than or equal to Funding Max"

Reply:
Got you, the validation messages